test/e2e-apiserver-test/suite_test.go

```diff
@@ -200,7 +200,6 @@ func delete(path string) *http.Response {
 	req.Header.Add("Authorization", token)
 	response, err := client.Do(req)
 	Expect(err).Should(BeNil())
-	defer response.Body.Close()
```

Comment:
why don't close?

Reply:
Here is a bug before. Out func should ready content from body. A closed body will generate a bug. Here is align with other (get,put,post) func

Reply:
when will the body closed? if no one close it, it will cause memory leak

Reply:
this func only used for apiserver-e2e-test.

Reply:
that's fine here, but we shouldn't make bad examples.

Reply:
Ok will close the body in out func.